### ISSUE TITLE
[vizbuilder] empty space on large width screens

### ISSUE THREAD

**Issue by davelandry** [Datawheel/canon#201]
when viewing the vizbuilder on a wide screen, it looks like there's a max-width placed on the visualization tiles. the tiles should always add up to 100% width, and just show more one wider screens (like showing 3 across in this example) using flex

<img width="2326" alt="screen shot 2018-10-01 at 5 15 32 pm" src="https://user-images.githubusercontent.com/1847581/46316303-d2161500-c59d-11e8-8561-99786ddbab3c.png">

**Comment by cnavarreteliz** [comment]
@davelandry I was exploring the code, and we set a special width dependents of size of the screen. Our idea was to avoid distortions between width and height in charts. What do you think? Do I remove `width: 33.333%` in wide screens? 

Other option is to guess the number of charts. If the number is 3 or more, to set `width: 33.333%`. Two charts, `width: 50%` and one chart with `width: 100%`.


```
@media screen and (min-width: 1200px) {
  .area-chart .multi .chart-card { width: 50%; }
}

@media screen and (min-width: 1700px) {
  .area-chart .multi .chart-card { width: 33.333%; }
}
```

**Comment by cnavarreteliz** [comment]
UPDATE: I'm testing with `:nth-child`

**Comment by davelandry** [comment]
@cnavarreteliz no guessing, use flexbox!

abstract example:

```css
.container {
  display: flex;
  flex-wrap: wrap;
  & .tile {
    flex: 1;
    height: 200px;
    min-width: 300px;
  }
}
```

**Comment by cnavarreteliz** [comment]
Closed via #208
